packages/wallet/src/components/PaymentPointerCard.tsx

```diff
@@ -157,7 +157,7 @@ export const PaymentPointerCard = ({
 }
 
 const cardStyles = `
-flex items-center justify-between border-b border-b-green-4 px-2 py-3
+flex items-center justify-between border-b border-b-green-4 px-2 pb-3 pt-1
 [&:nth-child(4n+1)_div_button]:bg-green-5 [&:nth-child(4n+1)_div_button:hover]:bg-green-6 
```

Comment:
there is a mt-8 for the payment pointer cards, that I could not overwrite :D  SO I added less padding to top then bottom, so that the cards look equaldistance from toop and bottom

Reply:
`space-y-2` adds some spacing between the cards. We can change this value and keep `py-3` for `PaymentPointerCard`.

https://github.com/interledger/testnet/blob/4432c4ce7b475ce3c5bb4c7f510b3c158aa15942/packages/wallet/src/pages/account/%5Bid%5D.tsx#L78-L85